cacher/accumulator.go

```diff
@@ -122,18 +122,23 @@ func (a *Accumulator) Append(sample *model.Sample) error {
 	return nil
 }
 
-// ChunksAfter returns a clone of all chunks in-time-ascending-order which contain
-// at least one value after the provided time. The result may be a length zero slice.
-func (a *Accumulator) ChunksAfter(after int64) []chunk.Chunk {
-	chunks := make([]chunk.Chunk, 0)
+// Chunks returns a slice of []byte representing each chunk that has data after
+// the specified timestamp.
+func (a *Accumulator) Chunks(after int64) ([][]byte, error) {
+	result := make([][]byte, 0)
```

Comment:
I don't think there's a functional difference. We haven't set a "styleguide" so there's no right answer from that perspective. I want a PR robot to do style checks. I don't think style is important enough for humans to spend much time on.